### PR TITLE
Fixes in progress

### PR DESCRIPTION
**Pull request by lexoyo**

### CODE REVIEW

**Comment by lexoyo** [comment]
@JbIPS can you confirm that these fixes are not needed anymore?
Thx

**Comment by JbIPS** [comment]
The `session.account_name` null check could be useful but not the path one

File: lib/unifile-fs.js [hunk 1]
```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
+    // is the path alreay resolved
+    if(path.startsWith(Os.homedir()) && Path.resolve(path).startsWith(Os.homedir()))
```
Comment (JbIPS) [inline lib/unifile-fs.js]: The FsConnector already have a `rootPath` property to resolve against. It defaults to the first sandbox path if given or `/`.

File: lib/unifile-fs.js [hunk 2]
```diff
@@ -32,6 +33,19 @@ class FsConnector {
     this.name = NAME;
   }
 
+  resolvePath(path) {
```
Comment (JbIPS) [inline lib/unifile-fs.js]: I'd rather use `Path.resolve(this.rootPath, path)`. Also, there is a function to check if the path is already resolved: `Path.isAbsolute(path)`

**Comment by JbIPS** [comment]
I'm sorry this is too old and already fixed on master. Thank you for the PR

**Pull request closed without merging**